docs/codeql/conf.py

```diff
@@ -50,9 +50,10 @@
 import sys
 
 def setup(sphinx):
+    import sphinx as sphinx_mod
```

Comment:
Nitpick recommendation, I would suggest to import the module at the top of the file, module-level instead of function-level, that way we will detect dependency problems (like the sphinx module not being available) earlier, at load time instead of function-calling time